components/infobox/wikis/rocketleague/infobox_person_player.lua

```diff
@@ -0,0 +1,263 @@
+---
+-- @Liquipedia
+-- wiki=rocketleague
+-- page=Module:Infobox/Person/Player
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Player = require('Module:Infobox/Person')
+local String = require('Module:StringUtils')
+local Class = require('Module:Class')
+local Earnings = require('Module:Earnings')
+local Namespace = require('Module:Namespace')
+local Variables = require('Module:Variables')
+local Page = require('Module:Page')
+local YearsActive = require('Module:YearsActive')
+
+local Injector = require('Module:Infobox/Widget/Injector')
+local Cell = require('Module:Infobox/Widget/Cell')
+local Title = require('Module:Infobox/Widget/Title')
+local Center = require('Module:Infobox/Widget/Center')
+
+local _BANNED = mw.loadData('Module:Banned')
+
+local _pagename = mw.title.getCurrentTitle().prefixedText
+
+local CustomPlayer = Class.new()
+
+local CustomInjector = Class.new(Injector)
+
+local _args
+
+local _START_YEAR = '2015'
+local _CURRENT_YEAR = mw.getContentLanguage():formatDate('Y')
+local _ROLE_CATEGORIES = {
```

Comment:
You could consider just doing a bunch of string.finds, then you don't need all the duplicate lines. For example:

if string.find('player') ~= nil then add 'Players' end

Reply:
good idea